tests/test_versioning.py

```diff
@@ -0,0 +1,18 @@
+from packaging.version import parse
+import dash_ag_grid as dag
+
+
+def test_vs001_versioning():
+    """
+    Test that the major and minor versions of bundled Ag Grid align with
+    the major and minor versions of the Python package.
+
+    See CONTRIBUTING.md for more information on versioning this package.
+    """
+    ag_version = parse(dag.grid_version)
+    dash_ag_version = parse(dag.__version__)
+
+    assert (
```

Comment:
Hm I guess we only would want to run this test when the version is bumped...

Reply:
That's why it needs to be a test before dist and not on github

Reply:
Touche.